doc/install.rst

```diff
@@ -93,5 +95,4 @@ The `WinPython <https://winpython.github.io/>`_ project distributes
 scikit-learn as an additional plugin.
 
 
-For installation instructions for particular operating systems or for compiling
-the bleeding edge version, see the :ref:`advanced-installation`.
+For installation instructions for more distributions see :ref:`install_by_distribution`. For compiling the development version from source, or building the package if no distribution is available for your architecture, see the :ref:`advanced-installation`.
```

Comment:
This is awkward because it's under a heading about third party distributions and says "see Third party distributions of scikit-learn". Rather, put this paragraph before Anaconda and WinPython, and alias the link to say "see other distributions".